spec/features/view_ip_address_readiness_spec.rb

```diff
@@ -0,0 +1,52 @@
+# TODO: don't require eveerything ever to fill in a form
+require 'features/support/not_signed_in'
+require 'features/support/sign_up_helpers'
+require 'features/support/errors_in_form'
+require 'features/support/activation_notice'
+require 'support/notifications_service'
+require 'support/confirmation_use_case'
+require 'timecop'
+
+describe 'even more requires' do
+  include_examples 'confirmation use case spy'
```

Comment:
You don't need this if you create users already confirmed. Eg: `create(:user, :confirmed, :with_organisation)`. If they are already confirmed when created, then the trigger to send a confirmation email is not called. Then you can remove:
```
require 'support/notifications_service'
require 'support/confirmation_use_case'
```

Reply:
Yep, just chucked these all in, going to remove them one-by-one as I clear it up.